index.html

```diff
@@ -45,9 +48,192 @@ <h2>Software Developer</h2>
                     <p>I can help you build a product , feature or website Look through some of my work and experience!
                         If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
                 </div>
+                <div class="hr-flex">
+                    <hr>
+                </div>
 
             </section>
+            <div>
+                <div class="box-flex lang-frame">
+
+                    <div class="box-size">
+                        <ul>
+                            <li>language</li>
+                            <li>javascript</li>
+                            <li>Ruby</li>
+                            <li>Html</li>
+                            <li>Css</li>
+                        </ul>
+                    </div>
+                    <div class="box-size">
+                        <ul>
+                            <li>frameworks</li>
+                            <li>Bootstrap</li>
+                            <li>Ruby on Rails</li>
+                            <li>RSPec</li>
+                            <li>CapyBara</li>
+                        </ul>
+                    </div>
+                    <div class="box-size">
+                        <ul>
+                            <li>skills</li>
+                            <li>codekit</li>
+                            <li>github</li>
+                            <li>codepen</li>
+                            <li>gitlb</li>
+                        </ul>
+                    </div>
+
+                </div>
+                <div class="hr-flex">
+                    <hr>
+                </div>
+                <div class="background-img"></div>
+
+                <div class="social-icon">
+                    <ul>
+                        <li><img src="./asset/Vector1.png" alt=""></li>
+                        <li><img src="./asset/‌linkedin.png" alt=""></li>
+                        <li><img src="./asset/Vectorthumb.png" alt=""></li>
+                        <li><img src="./asset/Vectortwitter.png" alt=""></li>
+                        <li><img src="./asset/Vectorm.png" alt=""></li>
+                    </ul>
+                </div>
+
+                <div>
+                    <div class="work-icon">
+                        <div><img src="./asset/Group_55.png" alt=""></div>
+                        <div>
+                            <p>work</p>
+                        </div>
+                    </div>
+
+                </div>
+            </div>
+            <section>
```

Comment:
- [ ] The main container of the projects should have a `display:grid`. Please kindly implement that in your css file as this is a requirement and it will also help you later to make the site responsive. You can assign a class attribute to  the section tag, and then give it a display of `grid` in the css file.